mysql-test/main/ssl_crl.test

```diff
@@ -7,7 +7,7 @@
 --exec $MYSQL --ssl-ca=$MYSQL_TEST_DIR/std_data/cacert.pem --ssl-key=$MYSQL_TEST_DIR/std_data/server-new-key.pem --ssl-cert=$MYSQL_TEST_DIR/std_data/server-new-cert.pem test -e "SHOW STATUS LIKE 'Ssl_version'"
 
 --echo # try logging in with a certificate in the server's --ssl-crl : should fail
-# OpenSSL 1.1.1a correctly rejects the certificate, but the error message is different
---replace_regex /ERROR 2013 \(HY000\): Lost connection to MySQL server at '.*', system error: [0-9]+/ERROR 2026 (HY000): TLS\/SSL error: sslv3 alert certificate revoked/
+# OpenSSL 1.1.1a and later releases correctly rejects the certificate, but the error message is different
+--replace_regex /(ERROR 2013 \(HY000\): Lost connection to MySQL server at '.*', system error: [0-9]+|ERROR 2026 \(HY000\): TLS\/SSL error: sslv3 alert certificate revoked)/ERROR 2026 (HY000): TLS\/SSL error: ssl\/tls alert certificate revoked/
```

Comment:
Its good. For future reference `replace_regex` can accept multiple find/replace regexes so it isn't needed to resort to extended regex.

I see the new test result is what openssl-3.2.0 generates which is a good reference.

No other tests hit this change.

Could you actually compile 10.4 with openssl-3.2.0? I thought only 10.5+ supported openssl-3+?

Can you accept a CLA or put a "I submit under content under BSDv3 license" comment?

Reply:
Hi @grooverdan, the copyright information is included in the pull request description as well as the commit message.

Reply:
The 10.4 /10.5 thing is a moot point anyway, this can only land in 10.5.